c3po/api/dto.py

```diff
@@ -0,0 +1,12 @@
+# Data Transfer Object- Responsible for carrying data between processes
+from flask import current_app
+from flask_restx import Namespace, fields
+
+
+class FeedDto:
+    api = Namespace('feed', description='Feed Related operations')
+    songObject = api.model('songObject', {
```

Comment:
Given that feed would generally have all the data about the song, and sending additional data would not cost much, as it would usually be text, I think it is fine if all the data of the songs is sent. 

Also, it would be imperative to have an infinite scrolable feed with filters/search bar. Thus, we would need end points to send the "next section" of the feed.

I will try making a mockup of the feed tonight :+1: